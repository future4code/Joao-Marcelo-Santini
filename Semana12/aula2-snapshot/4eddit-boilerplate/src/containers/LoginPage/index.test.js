import React from "react";
import { LoginPage } from "./index";
import renderer from "react-test-renderer";


describe("Testa actions do 4eddit", () =>{
    test("Snapshot", () => {
        const tree = renderer.create(<LoginPage />)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });

});