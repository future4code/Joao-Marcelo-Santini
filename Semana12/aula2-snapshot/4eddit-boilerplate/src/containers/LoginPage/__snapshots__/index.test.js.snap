// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testa actions do 4eddit Snapshot 1`] = `
<div
  className="sc-fzXfMD ehYKWH"
>
  <form
    className="sc-fzXfME czuMTT"
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
      >
        E-mail
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          name="email"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="email"
          value=""
        />
      </div>
    </div>
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
      >
        password
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          name="password"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="password"
          value=""
        />
      </div>
    </div>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Login
      </span>
    </button>
  </form>
</div>
`;
